# Bepaling van warmtecapaciteit van een onbekend materiaal


# Introductie

Onbekende materialen kunnen geïdentificeerd worden door hun eigenschappen te meten. Een van deze eigenschappen is de warmtecapaciteit. In dit practicum gaan we de warmtecapaciteit van een onbekend materiaal bepalen door middel van een calorimeter experiment. Daarbij wordt een bepaalde massa van het materiaal naar een bekende temperatuur gebracht waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaatst. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend.

# Theorie

De soortelijke warmte $c$ van een materiaal is gedefinieerd als de hoeveelheid warmte $Q$ die nodig is om de temperatuur $T$ van een kilogram van het materiaal met één graad Celsius (of één Kelvin) te verhogen:

$$
    c = \frac{Q}{m \Delta T}
$$ (eq:heat_capacity)

Waarbij $Q$ de hoeveelheid warmte in Joules is, $m$ de massa in kilogram is en $\Delta T$ de verandering in temperatuur is. Gegeven de wet van Black, die stelt dat de totale hoeveelheid warmte in een geïsoleerd systeem constant blijft, kunnen we de warmte die het onbekende materiaal verliest gelijkstellen aan de warmte die het water opneemt:

$$
    Q_{materiaal} = -Q_{water} 
$$ (eq:black)

wanneer we de massa's en de begintemperaturen van beide systemen kennen, maar slechts een van de twee soortelijke warmtes, kunnen we de onbekende soortelijke warmte berekenen. We combineren vergelijkingen [](#eq:heat_capacity) en [](#eq:black) om de volgende vergelijking te krijgen:

$$
    T_e = \frac{c_w m_w T_{w,b}+c_m m_m T_{m,b}}{c_w m_w + c_m m_m}
$$ (eq:combined)

Waarbij de subscripts $b$ en $e$ respectievelijk staan voor begintoestand en eindtoestand, $w$ voor water en $m$ voor het onbekende materiaal.

Bij metingen aan verschillende massa's van het onbekende materiaal en vervolgens een least square fit aan bovenstaande vergelijking kunnen we een precieze waarde voor de soortelijke warmte van het onbekende materiaal bepalen. Dat is, wanneer de warmtecapaciteit van bijvoorbeeld de beker te verwaarlozen is.


# Methode en materialen

## Ontwerp
De bovenstaande theorie wordt gebruikt om de soortelijke warmte van een onbekend materiaal te bepalen. Het experiment bestaat uit het verwarmen van verschillende massa's van het onbekende materiaal tot een bekende temperatuur, waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaats. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend. Om de tijd voor het meten van meerdere materialen te reduceren, worden de data van de verschillende groepen in het lokaal samengevoegd. Van tevoren is afgesproken welke massa's door welke groep worden gemeten, en hoeveel water er gebruikt wordt.

## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Calorimeter
- Thermometer of temperatuursensor
- Verwarmingsbron 
- Diverse massablokjes van onbekend materiaal
- Weegschaal
- Water
- Maatcilinder of maatbeker


```{figure} figures/c_onbekend_setup.jpg
:width: 70%
:label: fig_c_onbekend_setup

Een schematische weergave van de opstelling
```


## Procedure
Bespreek wie welke massa's van het onbekende materiaal gaat meten.
Bespreek ook hoeveel water er gebruikt gaat worden.
Bepaal de begintemperaturen.
Hevel het aantal afgesproken massa's in de maatbeker. 
Roer voorzicht zodat de temperatuur homogeen is.
Noteer de hoogste gemeten temperatuur, dit is $T_e$.
Wissel de metingen uit met de andere groepen en voer de data-analyse uit.

```{note}
Hieronder staat een voorbeeld hoe je een grafiek moet opslaan en vervolgens in je document kunt oproepen.
Het voorbeeld kun je natuurlijk straks verwijderen.
```

# Resultaten


In [2]:
# Water op kamertemperatuur: 21,7 graden
# temperatuur blokjes: 68,5 graden
# water na toevoeging blokjes: 24,7 graden

In [10]:
# Sla figuren op met  
# 
# plt.savefig("Figuren/naam.png", dpi=450)
# waarbij naam vervangen wordt door de bestandsnaam. 
# Onderstaande voorbeeld code en output grafiek 
import numpy as np
import matplotlib.pyplot as plt
from scipy.optimize import curve_fit

# Definiëren van de variabelen
Twb = 21.7 + 273.15 #K
Tmb = 68.5 + 273.15 #K
mb = 0.05*4 #kg
cw = 4186
mw = 0.301 #kg
Te = 24.7 + 273.15


# De vergelijking voor het bepalen van de onbekende soortelijke warmte
def soortelijk(Te):
    return ((Twb*cw*mw)-(Te*cw*mw))/((Te*mb)-(mb*Tmb))

print('De soortelijke warmte van het materiaal is', int(np.round(soortelijk(24.7+ 273.15))), 'J/kg K.')

# plek om metingen anderen toe te voegen
Twbarray = np.array([]) 
Tmbarray = np.array([])
mbarray = np.array([])
mwarray = np.array([])
Tearray = np.array([]) 

def eindtemp(Twb, c):
    return ((cw*mw*Twb + c*mb*Tmb)/(cw*mw+c*mb))

val, cov = curve_fit(eindtemp, Twbarray, Tearray)

print('De soortelijke warmte van het materiaal is', val[0], '+/-', cov[0])


De soortelijke warmte van het materiaal is 432 J/kg K.


ValueError: `ydata` must not be empty!

```{figure} figures/naam.png
:width: 50%
:label: fig_naam 

Hier is het onderschrift van de figuur.
```

# Discussie en conclusie

Hier een korte discussie en conclusie over de resultaten van het experiment en de implicaties daarvan.

# Discussie
De gevonden waarde voor de soortelijke warmte van het materiaal is 432 J/kg K. Het materiaal is onbekend, dus het is nog onduidelijk of deze strijdig is met de echte waarde of niet, maar aangezien het materiaal een vast materiaal was lijkt het in ieder geval in de goede ordegrootte te liggen. De meeste gewone metalen die de blokjes zouden kunnen zijn hebben een soortelijke warmte rondom deze waarde.


# Conclusie
Dit onderzoek is uitgevoerd met het doel om de soortelijke warmte van een onbekend materiaal te bepalen. Het materiaal wat gebruikt werd waren 4 metalen blokjes, deze hadden samen een massa van 200 gram. De massa's werden eerst opgewarmd tot een bekende temperatuur, waarna ze werden toegevoegd aan water op kamertemperatuur en erdoorheen geroerd. Uit een least squares fit volgt dat de soortelijke warmte van het onbekende materiaal gelijk is aan 432 J/kg K. De echte waarde is niet bekend, maar gezien het materiaal vast was en waarschijnlijk een soort metaal, lijkt deze waarde bij vergelijking met andere soortelijke warmtes van vaste metalen in de goede ordegrootte te liggen. Een vervolgonderzoek zou verbeterd kunnen worden door geleidelijker te roeren, door bijvoorbeeld de roerbeweging te mechaniseren, zodat de temperatuur meer homogeen is verdeeld.